www/src/utils/i18n.js

```diff
@@ -1,7 +1,11 @@
-const langs = require("../../i18n.json")
+const allLangs = require("../../i18n.json")
 const defaultLang = "en"
 
-const langCodes = langs.map(lang => lang.code)
+// Only allow languages defined in the LOCALES env variable
+// This allows us to test only languages that are "complete" or test only
+// a single language
+const langCodes = process.env.LOCALES ? process.env.LOCALES.split(" ") : []
+const langs = allLangs.filter(lang => langCodes.includes(lang.code))
```

Comment:
should we add a warning if one of the `LOCALES` they've entered does not appear in the `langCodes` list or is invalid?

Reply:
<img width="416" alt="console: fart is not a valid locale" src="https://user-images.githubusercontent.com/1278991/75182567-8b2a4180-56f5-11ea-924b-fac95c89a9b2.png">